test/integration/suite/run.go

```diff
@@ -26,7 +30,7 @@ func RunIntegrationSuite(t *testing.T, description string, clusterPrefix string)
 }
 
 var _ = SynchronizedBeforeSuite(func() []byte {
-	Ω(KindCreateCluster(ClusterName)).Should(Succeed())
```

Comment:
So sad. Omega gives the invaluable feel of the esoteric knowledge ;]

Reply:
Just...